<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongPlayerUi</class>
 <widget class="QWidget" name="SongPlayerUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;
border-width: 0px;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="7" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="infoLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Nimbus Sans L</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="7">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="7">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="7" alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>191</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QPushButton" name="nextSong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>101</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
        border-image: url(:/Resources/IconsOff/next.png);
}

QPushButton:pressed
{
        border-image: url(:/Resources/IconsOn/next.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="prevSong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>101</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
        border-image: url(:/Resources/IconsOff/previous.png);
}

QPushButton:pressed
{
        border-image: url(:/Resources/IconsOn/previous.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>191</height>
         </size>
        </property>
        <widget class="QPushButton" name="playButton">
         <property name="geometry">
          <rect>
           <x>29</x>
           <y>29</y>
           <width>132</width>
           <height>132</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
        border-style: solid;
        background: transparent;
        border-color: rgb(68,68,68);
        border-width: 5px;
        border-radius: 65px;
        background-image: url(:/Resources/IconsOff/play.png);
        background-position: center;
}

QPushButton:pressed
{
        border-style: solid;
        background: transparent;
        border-color: rgb(68,68,68);
        border-width: 5px;
        border-radius: 65px;
        background-image: url(:/Resources/IconsOn/play.png);
        background-position: center;
}

QPushButton:checked
{
        border-style: solid;
        background: transparent;
        border-color: rgb(68,68,68);
        border-width: 5px;
        border-radius: 65px;
        background-image: url(:/Resources/IconsOff/pause.png);
        background-position: center;
}

QPushButton:checked:pressed
{
        border-style: solid;
        background: transparent;
        border-color: rgb(68,68,68);
        border-width: 5px;
        border-radius: 65px;
        background-image: url(:/Resources/IconsOn/pause.png);
        background-position: center;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="geometry">
          <rect>
           <x>-2</x>
           <y>-9</y>
           <width>191</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent</string>
         </property>
         <layout class="QVBoxLayout" name="progressBarContainer">
          <property name="leftMargin">
           <number>1</number>
          </property>
         </layout>
        </widget>
        <widget class="QLabel" name="labelPic">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>130</width>
           <height>130</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/Resources/IconsOff/mask.png);
background: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="mask">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>130</width>
           <height>130</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/Resources/IconsOff/mask.png);
background: transparent; </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>labelPic</zorder>
        <zorder>mask</zorder>
        <zorder>verticalWidget</zorder>
        <zorder>playButton</zorder>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="loopButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border-image: url(:/Resources/IconsOff/repeat.png);
}

QPushButton:pressed
{
	border-image: url(:/Resources/IconsOn/repeat.png);
}

QPushButton:checked
{
	border-image: url(:/Resources/IconsOn/repeat.png);
}

QPushButton:checked:pressed
{
	border-image: url(:/Resources/IconsOff/repeat.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="shuffleButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border-image: url(:/Resources/IconsOff/shuffle.png);
}

QPushButton:pressed
{
	border-image: url(:/Resources/IconsOn/shuffle.png);
}

QPushButton:checked
{
	border-image: url(:/Resources/IconsOn/shuffle.png);
}

QPushButton:checked:pressed
{
	border-image: url(:/Resources/IconsOff/shuffle.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="9">
    <widget class="QSlider" name="volumeControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:vertical {
background: #5c91a1;
width: 4px;
position: absolute; 
}

QSlider::handle:vertical {
height: 16px;
border-radius: 8px;
background: grey;
margin: 0  -6px;
}

QSlider::add-page:vertical {
background: #5c91a1;
}

QSlider::sub-page:vertical {
background: white;
}</string>
     </property>
     <property name="sliderPosition">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="7">
    <widget class="QPushButton" name="playerToList">
     <property name="styleSheet">
      <string notr="true">background-color:blue;</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
